Profiles/cio-Symptom.xml

```diff
@@ -33,39 +33,20 @@
     <element id="Observation">
       <path value="Observation" />
       <short value="Symptom" />
-      <definition value="A phenomenon with which a condition in the patient presents itself." />
-      <comment value="This also includes a complaint or a finding, but not additional research, such as laboratory tests, function tests and imaging studies." />
+      <comment value="This profile is always used together with an instance conforming to [cio-Condition](http://nictiz.nl/fhir/StructureDefinition/cio-Condition) and [cio-Reaction](http://nictiz.nl/fhir/StructureDefinition/cio-Reaction). &#xD;&#xA;&#xD;&#xA;Note that the concept RelationCondition from the Symptom building block is not mapped within this profile. Instead, the relationship between Symptom and Condition is established through the `Condition.evidence.detail` element. Please refer to the [cio-Condition](http://nictiz.nl/fhir/StructureDefinition/cio-Condition) profile for more guidance." />
       <alias value="Symptoom" />
       <mapping>
-        <identity value="cio-dataset-200-beta2-20231214" />
-        <map value="cio-dataset-200-beta2-890" />
+        <identity value="cio-dataset-20-20231214" />
+        <map value="cio-dataset-20-890" />
```

Comment:
This should be 'cio-data**element**-...' (and similarly for all other concepts).

Reply:
fixed